htdocs/xsl/plus/layout.xsl

```diff
@@ -3,7 +3,9 @@
     xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xlink="http://www.w3.org/1999/xlink"
     xmlns:mml="http://www.w3.org/1998/Math/MathML"
     xmlns:math="http://www.w3.org/2005/xpath-functions/math"
-    xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl" exclude-result-prefixes="xs math xd"
+    xmlns:msxsl="urn:schemas-microsoft-com:xslt"
```

Comment:
Lo que ocurre es que realizo primero la limpieza del label en mixed-citation
```
        <xsl:variable name="rtf-mixed-citation">
            <xsl:choose>
                <xsl:when test="label">
                    <xsl:apply-templates select="mixed-citation" mode="without-label">
                        <xsl:with-param name="label" select="label"/>
                    </xsl:apply-templates>
                </xsl:when>
                <xsl:otherwise>
                    <xsl:copy-of select="mixed-citation"/>
                </xsl:otherwise>
            </xsl:choose>
        </xsl:variable>
```
Lo que regresa de nuevo el elemento mixed-citation, pero el valor de la variables es un RTF y no se le pueden aplicar nuevos templates, y agregue http://exslt.org/common para utilizar el rtf como node-set() http://www.xml.com/pub/a/2003/07/16/nodeset.html.
Y agregue esas lineas de más de microsoft que efectivamente no es necesaria, en un momento actualizo sin esa linea, la linea de oxygen ya estaba originalmente, la quito de una vez con la actualización?